web/assets/css/production.less

```diff
@@ -202,8 +202,10 @@ li.prep-status {
 .build-step .header {
   cursor: pointer;
   clear: both;
-  position: relative;
```

Comment:
@jomsie Sorry to pull your attention back to this - I realized we had a duplicate `position` tag here. Can you re-approve?